classes/Validate.php

```diff
@@ -61,8 +61,16 @@ public static function isIp2Long($ip)
         return preg_match('#^-?[0-9]+$#', (string) $ip);
     }
 
+    /**
+     * @deprecated since PrestaShop 8.1 and will be removed in Prestashop 9.0
```

Comment:
Ok to remove this one, but in this case you should remove its usage as well. Basically not setting a validation is equivalent as using this one.

The only use cases I could find are in `classes/order/OrderPayment.php` where you should remove all the `'validate' => 'isAnything',`